npm test: 7 failed tests in 'bem make'. v0.6.16, release-1.0.0

В 0.6.16 валятся исключительно bem make тесты.

```
Error: Cannot find module '../../bem-bl/blocks-common/.bem/level.js'
      at Object.<anonymous> (/home/alex/repos/bem-tools/test-make-temp/blocks/.bem/level.js:1:95)
```

И все, в основном, про это же.

В 1.0.0 не только они, но и они тоже. Все-таки alpha5, это понятно.

В общем, может быть я дурак, но банально не хватало `git submodule update --init`

Может лучше это внести в `make test`? ;-(

--- a/.travis.yml
+++ b/.travis.yml
@@ -10,4 +10,3 @@ matrix:
     - node_js: "0.11"
 
 before_install: "npm -g install npm"
-before_script: "ln -s .. node_modules/bem && ls -la node_modules && git submodule init && git submodule update"

--- a/GNUmakefile
+++ b/GNUmakefile
@@ -17,8 +17,13 @@ clean:
 jshint:
 	$(JSHINT) lib test
 
+.PHONY: test-setup
+test-setup:
+	@test -e node_modules/bem || ln -s .. node_modules/bem
+	@git submodule update --init
+
 .PHONY: test
-test: jshint
+test: test-setup jshint
 	$(MOCHA)
 
 .PHONY: lib-cov

